test/profiler/test_profiler.py

```diff
@@ -127,6 +128,26 @@ def custom_layer(input_ten):
             q = s.sum()
             q.backward()
 
+
+@unittest.skipIf(IS_WINDOWS, "Test is flaky on Windows")
+@unittest.skipIf(IS_MACOS, "Test is flaky on MacOS")
```

Comment:
Sorry, but is there an indication that test if flaky on any of those platforms?

Reply:
Hi @malfet  I haven't tested ITT on Windows and MacOS. However, from Intel(R) VTune Profiler perspective, all Windows, Linux and MacOS are supported. Shall I remove these 2 skipifs to have the UT running on Windows and MacOS as well?

Reply:
Yes, I think the only one needed here is `@unittest.skipIf(not HAS_PSUTIL, "Requires psutil to run")` and if test fails with ASAN, it should also be guarded against running on the platform

Reply:
The ITT PR actually doesn't depend on psutil, shall we just skip if the device is not a x86 platform?
By the way, any idea how I can find the keyword of x86? I tried to search "IS_*" in test folder, but didn't get a candidate.

Reply:
There is `IS_ARM64`, so you can do `@unittest.skip_if(IS_ARM64, "ITT is not available on ARM")`

Reply:
Got it. Thanks.

Reply:
Hi @malfet , Would you review my updated codes? Since the PR doesn't depend on psutils, I added 1 skipif to skip on ARM.

Reply:
`skipIf`,  not `skip_if`?

Reply:
correct to skipIf